common/test/TestDevice.cs

```diff
@@ -25,7 +25,7 @@ public enum ConnectionStringAuthScope
 
     public class TestDevice
     {
-        private const int DelayAfterDeviceCreationSeconds = 5;
+        private const int DelayAfterDeviceCreationSeconds = 0;
```

Comment:
Service used to have an issue, new registered device could be connected with not found error. Please confirm it's addressed.

Reply:
According to @simonporter we should not need to wait anymore right after device creation.
I've tested on my hub and Canary and WestUS proved that to be true, at least in the last runs.

I don't want to remove the code though: if this comes back, we can quickly increase the time back to something that works around the issue.